bitnami/etcd/README.md

```diff
@@ -262,6 +262,8 @@ The command removes all the Kubernetes components associated with the chart and
 | `disasterRecovery.cronjob.podAnnotations`       | Pod annotations for cronjob pods                                        | `{}`           |
 | `disasterRecovery.cronjob.resources.limits`     | Cronjob container resource limits                                       | `{}`           |
 | `disasterRecovery.cronjob.resources.requests`   | Cronjob container resource requests                                     | `{}`           |
+| `disasterRecovery.cronjob.nodeSelector`   | Cronjob container node selector                                    | `{}`           |
+| `disasterRecovery.cronjob.tolerations`                           | Tolerations for cronjob assignment                                                            | `[]`            |
```

Comment:
Please use the [Readme Generator For Helm](https://github.com/bitnami-labs/readme-generator-for-helm) to generate this in the proper format